llumnix/backends/vllm/llm_engine.py

```diff
@@ -31,20 +30,43 @@
 from llumnix.backends.backend_interface import BackendInterface
 from llumnix.backends.vllm.scheduler import SchedulerLlumnix
 from llumnix.backends.vllm.sequence import SequenceGroupLlumnix
-from llumnix.backends.vllm.utils import detect_unsupported_feature
 from llumnix.backends.profiling import LatencyMemData
 from llumnix.server_info import ServerInfo
 from llumnix.config import MigrationConfig
+from llumnix.rpc.queue_client import QueueClient
 
 
 logger = init_logger(__name__)
 
+@ray.remote(num_cpus=0)
+class AsyncActor:
+    def __init__(self, instance_id):
+        self.request_output_queue_client = QueueClient()
+        self.instance_id = instance_id
+        self.engine_actor_handle = None
+
+    async def put_nowait_batch_to_server(self,
```

Comment:
Do we really need to access the engine actor from inside the queue actor? Seems you use the engine actor just for aborting requests. If the queue actor is always invoked by the engine actor, then we can try-except the exception directly in the engine actor?

Reply:
no, since we do not use ray.get() when calling AsyncActor.put_nowait_batch_to_server, the exception would not be raised to LLMEngine.

Reply:
Now, I replace the async actor with thread instead, but the performance is nealy the same.